src/test/java/app/bpartners/api/integration/LocalInvoiceIT.java

```diff
@@ -321,11 +320,6 @@ void crupdate_invoice_ko() {
                 .quantity(1)
                 .vatPercent(1000));
     assertDoesNotThrow(first_ref_exec(api, firstInvoiceId, uniqueRef));
-    assertThrowsApiException(
```

Comment:
No. If this test must be deleted, then you need to change something in the implementation. 

Do not forget that, each users can specify his own reference, but the reference must not be ONLY generated auto.